packages/app/src/lib/import.ts

```diff
@@ -139,16 +147,18 @@ export async function importLegacyContainer(container: PBES2Container) {
     return Promise.all(items);
 }
 
-export function isPBES2Container(data: string) {
+export async function isPBES2Container(file: File) {
```

Comment:
I could've made this one check the `.pbes2` file extension, but this is a better check and reading a file as text is pretty cheap. Let me know if you disagree, here.